### PR TITLE
fix: uninitialized controller and repo

### PR DESCRIPTION
**Pull request by Azanul**
## Problem

Repository and Controller were not initialized

## Solution

Initialized a repository and attached it to APIHandler

### CODE REVIEW

File: handlers/resources_handler.go [hunk 1]
```diff
@@ -29,8 +30,10 @@ type ApiHandler struct {
 }
 
 func NewApiHandler(ctx context.Context, telemetry bool, analytics utils.Analytics, db *bun.DB, cfg models.Config, configPath string, accounts []models.Account) *ApiHandler {
+	repo := sqlite.NewRepository(db)
```
Comment (AvineshTripathi) [inline handlers/resources_handler.go]: what about postgres?

File: controller/controller.go [hunk 2]
```diff
@@ -32,7 +30,7 @@ type accountOutput struct {
 }
 
 type Repository interface {
-	HandleQuery(context.Context, repository.QueryType, interface{}, [][3]string) (sql.Result, error)
+	HandleQuery(context.Context, string, interface{}, [][3]string) (sql.Result, error)
```
Comment (AvineshTripathi) [inline controller/controller.go:33]: any reason for not using QueryType

Comment (Azanul) [reply controller/controller.go:33]: query titles are declared as string. I'm even considering dropping the type.

**Review by mlabouardy — APPROVED** [review]

File: handlers/resources_handler.go (hunk 1 above)
Comment (Azanul) [reply handlers/resources_handler.go]: fixed

**Review by AvineshTripathi — APPROVED** [review]

**Pull request merged**